gradle/publishing.gradle

```diff
@@ -67,15 +67,23 @@ publishing {
         }
         maven {
```

Comment:
We can now remove this step on publishing the artifact to private nexus repo.

Reply:
I decided to keep it, it is no harm there. Travis can chose which repo to publish to

Reply:
we can always remove it when we are sure that private nexus will not be used anymore